### PR TITLE
Demon Prison F2

### PR DESCRIPTION
**Pull request by vtange**

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -6978,22 +6978,22 @@ QUEST_LV_0200_20150428_006981	Velochard has been defeated and a treasure chest a
...
-QUEST_LV_0200_20150428_006998	Defeat Harukals in 4th Isolation Area
-QUEST_LV_0200_20150428_006999	You're prepared to defeat the Demon Lord Nuaele. Talk to Kupole Arune.
-QUEST_LV_0200_20150428_007000	Defeat the Demon Lord Nuaele
+QUEST_LV_0200_20150428_006985	$Kupole Audra asked for your help to defeat the demons that are performing rituals at the Rituala Assembly Area to help Demon Lord Blut escape.
+QUEST_LV_0200_20150428_006986	$You've interrupted the ritual for Blut's escape. Go back and report to Kupole Audra at the Corridor of Monitor.
+QUEST_LV_0200_20150428_006987	$Kupole Audra says you need to weaken Blut's influence. Steal Blut's Mark from the demons in Bjaurer Hideout.
+QUEST_LV_0200_20150428_006988	$Kupole Audra says the power gathered by Blut can only be controlled by Hauberk. Help Hauberk absorb the power of Blut from the Blut Altar in Concentrated Management Area.
+QUEST_LV_0200_20150428_006989	$Kupole Arune says you need to stem the power of Demon Lord Nuaele to stabilize the Goddess' barrier. Defeat Nuaele's subordinates.
+QUEST_LV_0200_20150428_006990	$Nuaele's subordinates are defeated. Go back and report to Kupole Arune.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Demon Lord Nuaele

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -6978,22 +6978,22 @@ QUEST_LV_0200_20150428_006981	Velochard has been defeated and a treasure chest a
...
+QUEST_LV_0200_20150428_006991	$Defeat the demons of the 1st Isolation Area
+QUEST_LV_0200_20150428_006992	$Some of Nuaele's demons used to be Hauberk's demons who've received empowered Marks from Hauberk before. Defeat Nuaele's demons in the 2nd Isolation Area and get Hauberk's Marks back.
+QUEST_LV_0200_20150428_006993	$Use the piece of the seal to suppress the demons in the 3rd Isolation Area and find out Nuaele's plans.
+QUEST_LV_0200_20150428_006994	$Found out Nuaele's plans. Tell Kupole Aurne about Nuaele's plans.
+QUEST_LV_0200_20150428_006995	$Figured out the plans of Nuaele. Talk to Kupole Arune on what to do next.
+QUEST_LV_0200_20150428_006996	$Kupole Arune suggested defeating the strongest and most loyal servants of Nuaele. Defeat the Harugals in the 4th Isolation Area.
+QUEST_LV_0200_20150428_006997	$Defeated the Harugals. Talk to Kupole Arune again.
+QUEST_LV_0200_20150428_006998	$Defeat Harugals in the 4th Isolation Area
+QUEST_LV_0200_20150428_006999	$You're prepared to defeat the Demon Lord Nuaele. Talk to Kupole Arune.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Remove 'the'.

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -6978,22 +6978,22 @@ QUEST_LV_0200_20150428_006981	Velochard has been defeated and a treasure chest a
...
+QUEST_LV_0200_20150428_006986	$You've interrupted the ritual for Blut's escape. Go back and report to Kupole Audra at the Corridor of Monitor.
+QUEST_LV_0200_20150428_006987	$Kupole Audra says you need to weaken Blut's influence. Steal Blut's Mark from the demons in Bjaurer Hideout.
+QUEST_LV_0200_20150428_006988	$Kupole Audra says the power gathered by Blut can only be controlled by Hauberk. Help Hauberk absorb the power of Blut from the Blut Altar in Concentrated Management Area.
+QUEST_LV_0200_20150428_006989	$Kupole Arune says you need to stem the power of Demon Lord Nuaele to stabilize the Goddess' barrier. Defeat Nuaele's subordinates.
+QUEST_LV_0200_20150428_006990	$Nuaele's subordinates are defeated. Go back and report to Kupole Arune.
+QUEST_LV_0200_20150428_006991	$Defeat the demons of the 1st Isolation Area
+QUEST_LV_0200_20150428_006992	$Some of Nuaele's demons used to be Hauberk's demons who've received empowered Marks from Hauberk before. Defeat Nuaele's demons in the 2nd Isolation Area and get Hauberk's Marks back.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:6988]: Nuaele's demons

Comment (vtange) [inline QUEST_LV_0200.tsv:6988]: I don't see any misspelling on 6992

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**